src/main/java/seedu/address/model/recipe/Recipe.java

```diff
@@ -120,7 +128,8 @@ public String toString() {
         // TODO: edit if needed
         StringBuilder sb = new StringBuilder();
         sb.append(getName())
-                .append(String.format("; Portions: %2f", getPortion()));
+                .append(String.format("; Completion time: %d (mins), Portions: %2f",
+                        getCompletionTime().value, getPortion()));
```

Comment:
minor suggestion follow-up:

```suggestion
                .append(String.format("; Completion time: %s, Portions: %2f",
                        getCompletionTime(), getPortion()));
```